docs/reference/commandline/tye-run.md

```diff
@@ -43,28 +43,25 @@ If a directory path is specified, `tye run` will default to using these files, i
 
     Write structured application logs to the specified log providers. Supported providers are console, elastic (Elasticsearch), ai (ApplicationInsights), seq.
 
-- `--dtrace <logs>`  
+- `--dtrace <logs>`
 
     Write distributed traces to the specified providers. Supported providers are zipkin.
 
 - `--debug <service>`
 
     Waits for debugger attach to service. Specify `*` to wait to attach to all services.
 
-- `--verbosity <verbosity>`
+- `-f|--framework <framework>`
 
-    Sets the output verbosity of the process.  
-    Possible values are 
-    
-    * `debug` - display all logs that the process outputs
-    * `info` - display only informational logs
-    * `quiet` - display only warnings and errors
+    The target framework hint to use for all cross-targeting projects with multiple TFMs. This value must be a valid target framework for each individual cross-targeting project. Non-crosstargeting projects will ignore this hint and the value TFM configured in tye.yaml will override this hint.
 
-    The default value is `info`
+- `-v|--verbosity <Debug|Info|Quiet>`
+
+    The verbosity of logs emitted by `tye run`. Defaults to Info.
```

Comment:
Why did you remove the values part?

Reply:
I'll revert it now I think about it. I originally made it to be more consistent with the other commands but I think the extra info is good.